colossalai/context/parallel_context.py

```diff
@@ -15,30 +15,16 @@
 
 from .parallel_mode import ParallelMode
 from .random import add_seed, get_seeds, set_mode
+from colossalai.context.singleton_meta import SingletonMeta
```

Comment:
Is "from .singleton_meta import SingletonMeta" better?

Reply:
Remember that you should generally opt for absolute imports over relative ones, unless the path is complex and would make the statement too long.

https://realpython.com/absolute-vs-relative-python-imports/

You can read Pytorch or tensorflow code. They soldom use relative path...

Reply:
ok